config/application.rb

```diff
@@ -31,9 +31,15 @@ class Application < Rails::Application
 
     if AppConfig.env.ruby_workers_enabled == 'true'
       config.active_job.queue_adapter = :delayed_job
+
+      FileUtils.mkdir_p(Rails.root.join('log'))
+      config.active_job.logger = ActiveSupport::Logger.new(Rails.root.join('log', 'workers.log'))
     else
       config.active_job.queue_adapter = :inline
     end
+
+    config.active_job.logger.formatter = config.log_formatter
```

Comment:
What is `config.log_formatter` here?

Reply:
It's either `Identity::Logging::DevelopmentLogFormatter` or `Identity::Logging::LogFormatter`, but I didn't feel like copy-pasting the logic

https://github.com/18F/identity-logging/blob/main/lib/identity/logging/railtie.rb#L11-L15

Reply:
👍🏼